dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -129,7 +130,19 @@ namespace winrt::Microsoft::Windows::PushNotifications::Helpers
 
     inline wil::com_ptr<INotificationsLongRunningPlatform> GetNotificationPlatform()
     {
-        return wil::CoCreateInstance<NotificationsLongRunningPlatform, INotificationsLongRunningPlatform>(CLSCTX_LOCAL_SERVER);
+        // If supported, allow use of class registrations from less trusted sources.
+        auto supportMarker{ wil::CoCreateInstanceNoThrow<ISupportAllowLowerTrustActivation>(CLSID_ActivationCapabilities) };
+        if (supportMarker)
+        {
+            return wil::CoCreateInstance<INotificationsLongRunningPlatform>(CLSID_NotificationsLongRunningPlatform, CLSCTX_LOCAL_SERVER |
+                CLSCTX_ALLOW_LOWER_TRUST_REGISTRATION);
+        }
+        else
+        {
+            // Even if we can't pass CLSCTX_ALLOW_LOWER_TRUST_REGISTRATION, we still want to attempt the activation.
+            // This might fail to find the class when run elevated on a downlevel SKU, but it will work otherwise.
+            return wil::CoCreateInstance<INotificationsLongRunningPlatform>(CLSID_NotificationsLongRunningPlatform, CLSCTX_LOCAL_SERVER);
```

Comment:
Should IsSupported() return false for this case?

Reply:
Would it make sense for IsSupported to also make this call?

Reply:
Need to discuss how to address this in another PR

Reply:
Tracking issue: https://github.com/microsoft/WindowsAppSDK/issues/3063